src/services/symbolDisplay.ts

```diff
@@ -64,6 +64,7 @@ namespace ts.SymbolDisplay {
         if (flags & SymbolFlags.SetAccessor) return ScriptElementKind.memberSetAccessorElement;
         if (flags & SymbolFlags.Method) return ScriptElementKind.memberFunctionElement;
         if (flags & SymbolFlags.Constructor) return ScriptElementKind.constructorImplementationElement;
+        if (flags & SymbolFlags.Signature) return ScriptElementKind.indexSignatureElement;

```

Comment:
`SymbolFlags.Signature` is also used for call and construct signatures. If tests are passing this _might_ be fine, but I would want to understand why you don’t have to worry about call and construct signatures here.

Reply:
I am actually not sure about this. The function returns ScriptElementKind.memberVariableElement ('property') for `SymbolFlags.Property` which is the other example I was looking at, I want to return ScriptElementKind.indexSignatureElement ('index') for SymbolFlags.Signature which is why I added this here, but I wasn't aware of SymbolFlags.Signature also being used for call and construct signatures.